Mobile Buy SDK/Mobile Buy SDK/Data/BUYClient+Checkout.h

```diff
@@ -220,6 +220,13 @@ typedef void (^BUYDataGiftCardBlock)(BUYGiftCard * _Nullable giftCard, NSError *
  */
 - (NSOperation *)removeGiftCard:(BUYGiftCard *)giftCard fromCheckout:(BUYCheckout *)checkout completion:(BUYDataCheckoutBlock)block;
 
+/**
+ *	Removes the discount from the checkout.
```

Comment:
nitpick but spacing is off in this comment compared to other comments in the file